lib/scripts/createGenesisDao.ts

```diff
@@ -110,18 +110,24 @@ export class GenesisDaoCreator {
 
     let avatarInst;
 
-    /**
-     * save info for later steps
-     */
     while (!avatarInst) {
```

Comment:
Can't you just wait using `TransactionService`?

Reply:
I thought about that.  Thing is, I'm not sure why it is necessary in the first place.  Truffle normally waits until something is mined before returning from the promise.  So why is the avatar not available?  I don't know, but I don't think TxService.watchForMined would help.

Reply:
Just a note:  I've generally been avoiding using Arc.js services unless really needed, just to keep things simple and clear in the deployment script.  In this case I'd feel justified in using it, but I'm not convinced it would help, and didn't feel it was important enough to spend $$$ on testing one way or the other, since this code works.

Reply:
What was the problem with the original code?

Reply:
was missing an `await` and was not catching the exception when thrown by `.at`